src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -207,7 +207,7 @@ const createCustomButtons = (settings) => {
         'data-se': customButton.dataAttr
       },
       className: customButton.className + ' default-custom-button',
-      title: customButton.title,
+      title: customButton.title || loc(customButton.i18nKey, 'login'),
```

Comment:
Curious - do we plan to support arguments for custom i18n keys? Would be great if we could be flexible and allow developers to optionally pass through positional arguments.

Reply:
Looks like it's listed in readme
https://github.com/okta/okta-signin-widget#custom-buttons

Reply:
Spoke offline - positional arguments isn't supported nor have been requested.